src/async-wrap.cc

```diff
@@ -76,6 +76,8 @@ static void Initialize(Handle<Object> target,
 Handle<Value> AsyncWrap::MakeCallback(const Handle<Function> cb,
                                       int argc,
                                       Handle<Value>* argv) {
+  if (!env()->CanCallIntoJs())
+    return Undefined(env()->isolate());
```

Comment:
Can you explain what's going on here? I don't understand why it should return early if `worker_context_ == nullptr`.